pet-pics.md

```diff
@@ -2,3 +2,8 @@
 
 This is my little old schnauzer, Pepper:
 ![Pepper](pet-pics/Pepper.png)
+
+---
+
+[@danieljustus32](https://github.com/danieljustus32)'s partner in crime and eating, Jasper:
```

Comment:
Hi @danieljustus32, I just wanted to try out commenting on a PR. Everything looks fine to me and you sure have a lovely cat! 🐱 
I'm sure @kapunahelewong will do a review any time soon. 😊

Reply:
Thanks @sambogaert! 😸

Reply:
Sorry, @danieljustus32 and @sambogaert - I totally missed the notification! But yes, this is lovely!